modules/bullet/space_bullet.h

```diff
@@ -194,6 +195,7 @@ class SpaceBullet : public RIDBullet {
 	void check_ghost_overlaps();
 	void check_body_collision();
 
+public:
```

Comment:
What's the use case for exposing this if the methods that use it are still private?

Reply:
The idea is that you can use this struct outside the `BulletPhysicsServer`, while implementing another feature for example. In this way it's not needed to duplicate this struct.